client/my-sites/stats/stats-purchase/stats-purchase-single-item.tsx

```diff
@@ -132,8 +152,10 @@ Thanks\n\n`;
 	return (
 		<>
 			<h1>{ translate( 'Jetpack Stats' ) }</h1>
-			<p>{ translate( 'The most advanced stats Jetpack has to offer.' ) }</p>
-			<StatsBenefitsCommercial />
+			{ ! isCommercialOwned && (
```

Comment:
Would it be clearer to consolidate the `! isCommercialOwned` check for only once?
```suggestion
			{ ! isCommercialOwned && (
				<>
					<p>{ translate( 'The most advanced stats Jetpack has to offer.' ) }</p>
					<StatsBenefitsCommercial />
				</>
			) }
			{ isCommercialOwned && <StatsUpgradeInstructions /> }
```